src/providers/postgres/qgspostgresconn.h

```diff
@@ -310,6 +310,13 @@ class QgsPostgresConn : public QObject
 
     QString connInfo() const { return mConnInfo; }
 
+    /**
+     * Returns the underlying database.
+     *
+     * \since QGIS 3.0
+     */
+    QString currentDatabase();
```

Comment:
const?

Reply:
It cannot be `const` because `currentDatabase()` uses the `PQexec` method which is not `const` itself. Unless there's another way to run the underlying query?

Reply:
what's your opinion on making PQexec const or adding a const version?

Reply:
Good point, the simpler the better...